client/src/components/ImageSlides.jsx

```diff
@@ -0,0 +1,13 @@
+const ImageSlides = ({ url }) => {
+  const styles = {
+    backgroundImage: `url(${url})`,
+    backgroundSize: 'cover',
+    backgroundPosition: 'center'
+  };
+
+  return (
+    <div className="image-slide" style={styles}></div>
+  );
+}
+
+export default ImageSlides
```

Comment:
choose how you want to style your component -- css component or css module. Avoid having your style script in the same file as the component.